src/polynomials/ngcd.jl

```diff
@@ -266,7 +266,7 @@ function ngcd(p::PnPolynomial{T,X},
 
     nr, nc = size(Sₓ) # m+1, m-n+2
     F = qr(Sₓ)
-    Q[1:nr, 1:nr] .= F.Q
+    copyto!(view(Q, 1:nr, 1:nr), F.Q)
```

Comment:
In case you need good performance on Julia v1.6, this line could be replaced by
```suggestion
    dest = view(Q, 1:nr, 1:nr)
    copyto!(dest, I)
    lmul!(F.Q, dest)
```
which is the approach taken in https://github.com/JuliaLang/julia/pull/39533/files.